lib/scenic/view.rb

```diff
@@ -22,15 +22,20 @@ class View
     # @return [Boolean]
     attr_reader :materialized
 
+    # True if the view is materialized with NO DATA
+    # @return [Boolean]
+    attr_reader :no_data
+
     # Returns a new instance of View.
     #
     # @param name [String] The name of the view.
     # @param definition [String] The SQL for the query that defines the view.
     # @param materialized [String] `true` if the view is materialized.
-    def initialize(name:, definition:, materialized:)
+    def initialize(name:, definition:, materialized:, no_data: false)
```

Comment:
I'm not sure `Scenic::View` needs to do anything with `no_data`. As best I can tell, we only use this construct when dumping the schema. At that point, I don't believe we know if the view was created with no data or not? Indeed, the only call site for `View.new` does not add the `no_data` parameter at all.

This effectively means that views created by loading schema would never use `WITH NO DATA`. It would be a migration-time only option.

This makes me a little nervous because theoretically that could allow production and development inconsistencies. For example, if in my migration I do `create_view :searches, materialized: { no_data: true }` and run that in development and production I'd be left with an empty `searches` view. If in development I later run `rake db:schema:load` then my view will now be populated in development only which may lead to some confusion.

Should we be concerned about that? Thoughts, @calebthompson @serg-kovalev ?